src/celscript/lua/celx_observer.cpp

```diff
@@ -284,21 +287,21 @@ static int observer_gototable(lua_State* l)
 
     lua_pushstring(l, "startInterpolation");
     lua_gettable(l, 2);
-    jparams.startInterpolation = celx.safeGetNumber(3, NoErrors, "", 0.25);
+    jparams.startInterpolation = celx.safeGetNumber(3, NoErrors, "", defaultStartInter);
     lua_settop(l, 2);
 
     lua_pushstring(l, "endInterpolation");
     lua_gettable(l, 2);
-    jparams.endInterpolation = celx.safeGetNumber(3, NoErrors, "", 0.75);
+    jparams.endInterpolation = celx.safeGetNumber(3, NoErrors, "", defaultEndInter);
     lua_settop(l, 2);
 
     lua_pushstring(l, "accelTime");
     lua_gettable(l, 2);
-    jparams.accelTime = celx.safeGetNumber(3, NoErrors, "", 0.5);
+    jparams.accelTime = celx.safeGetNumber(3, NoErrors, "", o->getDefaultAccelerationTime());
     lua_settop(l, 2);
 
     jparams.duration = max(0.0, jparams.duration);
-    jparams.accelTime = min(1.0, max(0.1, jparams.accelTime));
+    jparams.accelTime = celmath::clamp(jparams.accelTime, Observer::defaultAccelTimeMin, Observer::defaultAccelTimeMax);
     jparams.startInterpolation = min(1.0, max(0.0, jparams.startInterpolation));
```

Comment:
could you update to clamp here and on the next line as well?

Reply:
done